src/module.cpp

```diff
@@ -979,6 +979,61 @@ std::array<module::with_inputs, 3> module::split(const std::vector<instruction_r
     return {{std::move(mods1[0]), std::move(mods2[0]), std::move(mods2[1])}};
 }
 
+static void insert_params(module& m,
+                          const std::vector<instruction_ref>& inputs,
+                          std::unordered_map<instruction_ref, instruction_ref>& map_ins)
+{
+    auto n = m.get_parameter_shapes().size();
+    for(auto input : inputs)
+    {
+        if(contains(map_ins, input))
+            continue;
+        map_ins[input] = m.add_parameter(param_name(n++), input->get_shape().as_standard());
+    }
+}
+
+std::vector<instruction_ref>
```

Comment:
```suggestion
// General utility for fusing modules.  Inserts instructions into an instruction list.

std::vector<instruction_ref>
```

Reply:
There is already a description of the function in the header file.

Reply:
In `module.hpp`?  `insert_params` isn't even mentioned in that header.

Reply:
I thought it was for the `fuse` method. Let me add a description for that.